### ISSUE TITLE
Opaque Pre Pass Textures not Rendering Properly on Mobile Devices

### ISSUE THREAD

**Issue by TrevSaysHi** [godotengine/godot#39027]
**Godot version:**
3.2.1 -3.2.2 Beta 2
**OS/device including version:**
iOS 13.5, 
Android
**Issue description:**
When adding an opaque pre-pass texture to a mesh it does not render properly on mobile devices.
**Steps to reproduce:**
Add a transparent texture to a mesh instance and set it to opaque pre pass.
Expected result:
<img width="487" alt="Screen Shot 2020-05-24 at 9 27 08 PM" src="https://user-images.githubusercontent.com/65880004/82772997-82393100-9e06-11ea-8073-f68c6e937032.png">
Actual Result:
![IMG_89D042D2DEF6-1](https://user-images.githubusercontent.com/65880004/82773011-882f1200-9e06-11ea-97c7-23c8fc025bb3.jpeg)

**Minimal reproduction project:**
[OpaqueIssue](https://github.com/TrevSaysHi/OpaqueIssue)

**Comment by clayjohn** [comment]
Looks like the same issue as https://github.com/godotengine/godot/issues/36669

Funny enough, that one appears on Windows, not just Android and iOS.

**Comment by TrevSaysHi** [comment]
Yeah I do think it is a similar issue, when I applied the fix in the other issue it does now fully appear; however, there are still some issues with the leaves of the tree having a hard time rendering in front of each other.
Here's a new screen of the fix in the full app not just the minimal project.
![IMG_0207](https://user-images.githubusercontent.com/65880004/82822584-a2e7a200-9e6b-11ea-9df8-96ccd1e21aae.PNG)

**Comment by tinmanjuggernaut** [comment]
Probably fixed by #50721.
